data/json/professions.json

```diff
@@ -735,7 +735,7 @@
     "type": "profession",
     "ident": "soldier",
     "name": "Military Recruit",
-    "description": "You were a high school drop-out with one goal in mind: to join the military.  You finally got in, just in time for your training to get interrupted by a national emergency.  As far as you can tell, military command abandoned you in this hellhole when you missed the emergency evac.",
+    "description": "You were a high school drop-out with one goal in mind: to join the military.  You finally got in, just in time for your training to get interrupted by some sort of national emergency.  As far as you can tell, military command abandoned you in this hellhole when you missed the emergency evac.",
```

Comment:
I'd consider dropping the specification that the character is a high school drop-out, and just say that they'd had the goal of joining the military in mind for some period of time.